crates/astria-composer/src/config/searcher.rs

```diff
@@ -0,0 +1,119 @@
+use std::{
+    net::{
+        AddrParseError,
+        SocketAddr,
+    },
+    str::FromStr,
+};
+
+use figment::{
+    providers::{
+        Env,
+        Serialized,
+    },
+    Figment,
+};
+use serde::{
+    Deserialize,
+    Serialize,
+};
+use thiserror::Error;
+
+use super::cli;
+use crate::types::rollup::ChainId;
+
+const DEFAULT_SEQUENCER_URL: &str = "127.0.0.1:1317";
+const DEFAULT_API_URL: &str = "127.0.0.1:8080";
+const DEFAULT_CHAIN_ID: &str = "ethereum";
```

Comment:
changed the default value to `rolluptest`.

some clarification: this is the chain_id value that will go into the [`SecondaryAction` tx](https://github.com/astriaorg/astria/blob/noot/secondary-txs/crates/astria-proto/proto/astria/sequencer/v1/transaction.proto#L36) that the composer will be submitting to the sequencer. When I continue with implementing the read & write operations this will probably be moved to a `RollupConfig` struct that'll hold the execution node's rpc addr for reading, the chain_id to write to and whatever other config will be needed. so it will eventually be `RollupConfig::chain_id` or something like that.

if you have any name suggestions i'm not at all attached, but this is mostly a placeholder for now.